### ISSUE TITLE
Toggle per display

### ISSUE THREAD

**Issue by petermeierrr** [res2k/HDRTray#7]
I'm two displays usually. One big gaming monitor, and a secondary, desktop only screen. However, the latter also support HDR (somewhat) and toggles in parallel with the tray switch. That's quite undesirable.
In a perfect world, it would switch the monitor the tray icon is used on. But being able to hard-select which monitor will toggle would also be great.

**Comment by groooooooooooot** [comment]
great suggestion

**Comment by jmachado45** [comment]
I would like this as well!

**Comment by res2k** [comment]
I've added support for display selection in a prerelease: https://github.com/res2k/HDRTray/releases/tag/v0.5.90
I've opted to make it a "prerelease" only because selected displays currently can only be configured via HDRCmd (but the selection also applies to HDRTray!), and also b/c I'd like to collect a bit of feedback - ie does it work for you.
So please give the new build a spin.

**Comment by groooooooooooot** [comment]
the cmd line toggle for multiple displays seems to work for me (so far - knock on wood).  Expose it via the UI at some point and it'll  be great

**Comment by petermeierrr** [comment]
Thanks a lot for working on this!

fI tried it now, and it does make the tool usable as hoped. Finally that fake-HDR screen stays unchanged. It's great, really. I noticed that it does reinitialize with the tool, but not changing the mode. Ideally it wouldn't re-initialize at all if set to off from toggling, but that's just a "nice to have".

Another thought: sometimes I have a third screen connected (TV), but I may want to turn only one of the screens one.
Ideally there would be a direct selection which screen to turn HDR on for. It's easy enough to script different configurations with the cmd line tool, so I don't really mind, but I'm curious hwo the selection will be integrated into the tray app.

Again, thank you for this update. If you could see the grin on my face right now. :D

**Comment by res2k** [comment]
> fI tried it now, and it does make the tool usable as hoped. Finally that fake-HDR screen stays unchanged. It's great, really. I noticed that it does reinitialize with the tool, but not changing the mode. Ideally it wouldn't re-initialize at all if set to off from toggling, but that's just a "nice to have".

I think that's Windows' doing. Just toggling HDR for a single display also affects other displays; not sure I can do anything about it.

**Comment by andriy-kudrya** [comment]
Hi! Thanks for the tool. I've tried this build with my monitors. Selective HDR works well for me thus far.

**Comment by res2k** [comment]
Happy between-the-years everyone!

I have something for you: A usable graphical configuration utility!
Find the release here: https://github.com/res2k/HDRTray/releases/tag/v0.5.91

You can either launch the configuration utility from the tray icon, or by running `HDRTrayConfig.exe`.
Please let me know how it works for you, or if you notice something behaving unexpectedly.

Note you'll need the Windows App SDK runtime 1.7 installed.
If this is not present there should be a dialog offering to take you to it's download page.
If that shouldn't work for some reason, find it here: https://learn.microsoft.com/en-us/windows/apps/windows-app-sdk/downloads (as of this writing, 1.7 is still "experimental" can be found further down the page.)

**Comment by andyhoeung** [comment]
Since Win + Alt + B toggles HDR for both of my monitors and I only want that on my main, Game Bar's 'use HDR on main display' doesn't work, and neither does removing the HDR metadata on my 2nd display. This has been the only solution that works for me. Legend!
